ethbloom/Cargo.toml

```diff
@@ -11,7 +11,8 @@ repository = "https://github.com/paritytech/primitives"
 [dependencies]
 tiny-keccak = "1.4"
 crunchy = { version = "0.1.6", features = ["limit_256"] }
-fixed-hash = { version = "0.2", default_features = false }
+rustc-hex = { version = "2.0" }
```

Comment:
Do we really need this? :/

Reply:
I don't know atm. I will look into it but wanted to keep the breakage of dependent crates as small as possible to be able to refocus my work as soon as possible.

Reply:
But `rustc-hex` was added in this PR, yes?

Reply:
Yes it was added. I will investigate.